cli/ballerina-cli/src/main/resources/create_cmd_templates/lib/Module.md

```diff
@@ -0,0 +1,5 @@
+Prints "Hello World!" with a main function.
```

Comment:
better to describe why we need Module.md and what we should add to it. we can even link the ballerina.io. we can add the descriptions as md comments